lightkurve/targetpixelfile.py

```diff
@@ -1130,6 +1131,124 @@ def cutout(self, center=None, size=5):
+        ax : `~matplotlib.axes.Axes`
+            A matplotlib axes object to plot into. If no axes is provided,
+            a new one will be generated.
+        normalized : bool
+            Default: False; if True, the normalized light curves will be plotted.
+        periodogram : bool
+            Default: False; if True, periodograms will be plotted, using normalized light curves.
+            Note that this keyword overrides normalized.
+        aperture_mask : ndarray or str
+            Highlight pixels selected by aperture_mask.
+            Only `pipeline`, `threshold`, or custom masks will be plotted.
+            `all` and None masks will be ignored.
+        style : str
+            Path or URL to a matplotlib style file, or name of one of
+            matplotlib's built-in stylesheets (e.g. 'ggplot').
+            Lightkurve's custom stylesheet is used by default.
+        kwargs : dict
+            e.g. `threshold` to be passed to create_threshold_mask
+        
+        """
+
+        if style == 'lightkurve' or style is None:
+            style = MPLSTYLE 
+        
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+
+            # get an aperture mask for each pixel
+            masks = np.zeros((self.shape[1]*self.shape[2], self.shape[1], self.shape[2]), dtype='bool')
+            for i in range(self.shape[1]*self.shape[2]):
+                masks[i][np.unravel_index(i, (self.shape[1], self.shape[2]))] = True
+            
+            pixel_list = []
+
+            for j in range(self.shape[1]*self.shape[2]):
+                lc = self.to_lightcurve(aperture_mask=masks[j])
+                lc_norm = lc.normalize().remove_outliers()  
+                if periodogram == True:
+                    try:
+                        pixel_list.append(lc_norm.to_periodogram())
+                    except IndexError:
+                        pixel_list.append(None)
+                else:
+                    if normalized == True:
+                        if len(lc_norm.remove_nans().flux) == 0:
+                            pixel_list.append(None)
+                        else:
+                            pixel_list.append(lc_norm)
+                    elif normalized == False:
+                        if len(lc.remove_nans().flux) == 0:
+                            pixel_list.append(None)
+                        else:
+                            pixel_list.append(lc)
+
+        if aperture_mask == 'pipeline':
+            mask = self.pipeline_mask
+        elif aperture_mask == 'threshold':
+            mask = self.create_threshold_mask(**kwargs)
+        elif aperture_mask == 'all' or aperture_mask == None:
+            mask = np.zeros((self.shape[1], self.shape[2]), dtype='bool')
+        else:
+            mask = aperture_mask
+
+        with plt.style.context(style):
+            if title is None:
+                title = 'Target ID: {}, Quarter: {}'.format(self.targetid, self.quarter)
```

Comment:
K2 and TESS data don't have a quarter number, so I suggest showing only the target id:
```suggestion
                title = f'Target ID: {self.targetid}'
```
(Pro-tip: now that Lightkurve is Python 3 only, we can use f-strings as shown above, which are awesome!)